packages/eas-cli/src/build/android/applicationId.ts

```diff
@@ -5,7 +5,7 @@ import chalk from 'chalk';
 import fs from 'fs-extra';
 import path from 'path';
 
-import log from '../../log';
+import Log from '../../log';
```

Comment:
I dont have a strong preference but I think lowercase log is more common in CLI related tools, also this was lifted from expo-cli so it might make sense to keep them consistent (even if that means updating it over there).

Reply:
I think it depends on what is being exported. I made it a class so exporting uppercase made the most sense to me. In the old version it exported a function so lowercase made sense.

As a completely different approach to this PR I'd be okay removing the default export from the module entirely and just have all exports be named exports, but for some reason I like the android style `Log.level` a little bit better than naked `warn` calls.

Reply:
We could also call the module something like `Stdout`.